sktime/annotation/stray.py

```diff
@@ -0,0 +1,283 @@
+# -*- coding: utf-8 -*-
+"""Tests for STRAY (Search TRace AnomalY) outlier estimator."""
+
+import warnings
+
+import numpy as np
+from scipy.stats import iqr
+from sklearn.neighbors import NearestNeighbors
+
+from sktime.transformations.base import BaseTransformer
+
+__author__ = ["KatieBuc"]
+__all__ = ["STRAY"]
+
+
+def unitize(x):
+    diff = max(x) - min(x)
+    if diff == 0:
+        return np.zeros(len(x))
+    return (x - min(x)) / diff
+
+
+def standardize(x):
+    return (x - np.median(x)) / iqr(x)
+
+
+class STRAY(BaseTransformer):
```

Comment:
only because of the convenience of tags in `BaseTransformer`, specifically:
```
"X_inner_mtype": "np.ndarray"
```

currently not implemented in `BaseSeriesAnnotator`